rspotify-model/src/track.rs

```diff
@@ -59,6 +59,8 @@ pub struct FullTracks {
 /// relinking is applied.
 #[derive(Clone, Debug, Serialize, Deserialize, PartialEq, Eq)]
 pub struct SimplifiedTrack {
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub album: Option<SimplifiedAlbum>,
```

Comment:
So does it mean, when this field is none, Spotify just doesn't return this field rather than return this field with a null value?

Reply:
> So does it mean, when this field is none, Spotify just doesn't return this field rather than return this field with a null value?

Correct.  I'm referencing the album track endpoint which has no album within the track (I'm assuming it's because you're using the album to get the track):
https://developer.spotify.com/documentation/web-api/reference/get-an-albums-tracks

Reply:
Get it, thanks for your contribution. Merged :)